src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
You should use fallback fonts - alternative font-family in case the main one doesn't work. For example, you could use `font-family: Roboto, Arial, sans-serif;`